### PR TITLE
Add diagram and text to describe the demo

### PR DESCRIPTION
**Pull request by dargilco**
## Purpose

Add diagram and text to describe the custom command demo & visualization.
To see the diagram image, click on "display the ritch diff" icon on the .png file that got added (this is not the default view)

## Does this introduce a breaking change?

```
[ ] Yes
[x ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x ] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: custom-commands/hospitality/deployment/README.md [hunk 1]
```diff
@@ -10,8 +10,25 @@
...
+
+The following block diagram describes how the demo runs: 
+<p align="center">
+<img src="images/block-diagram.png"/>
+</a>
+</p>
+We assume that you read about Custom Commands and Speech SDK client applications, and here we describe the visualization part:
+
+* An Azure storage is created, which holds an HTML file and associated images needed to visualize all the states of the scene 
+* Azure storage also includes a table with one row, which is used to save the state of the scene
```
Comment (chschrae) [inline custom-commands/hospitality/deployment/README.md:24]: initially this table doesn't exist until the azure function is called. Not sure if you want to mention that or not.

File: custom-commands/hospitality/deployment/README.md [hunk 2]
```diff
@@ -10,8 +10,25 @@
...
+<p align="center">
+<img src="images/block-diagram.png"/>
+</a>
+</p>
+We assume that you read about Custom Commands and Speech SDK client applications, and here we describe the visualization part:
+
+* An Azure storage is created, which holds an HTML file and associated images needed to visualize all the states of the scene 
+* Azure storage also includes a table with one row, which is used to save the state of the scene 
+* You can view the HTML file in your web browser, since it will have a unique URL. It will show the default scene at first
+* An Azure function is created. It has access to the storage. When your Custom Command project has completed a dialog (e.g. "turn on the lights"), its completion rule will make a web call invoking the Azure function with query URL parameters indicating the desired new state of the scene (lights are on)
```
Comment (chschrae) [inline custom-commands/hospitality/deployment/README.md:26]: This sounds like you are saying the state is in the query URL parameters which I believe is not true. The state gets passed in the body where the parameter is the room name

Comment (dargilco) [reply custom-commands/hospitality/deployment/README.md:26]: Thanks, will fix

File: custom-commands/hospitality/deployment/README.md (hunk 1 above)
Comment (dargilco) [reply custom-commands/hospitality/deployment/README.md:24]: I was debating that (trying to keep it simple), but I can make some changes to indicate the table does not exist at the begining.  Thanks!

**Pull request merged**